tests/test_drs.py

```diff
@@ -0,0 +1,176 @@
+import json
+import tests.conftest
+import requests
+import responses
+from tests.default_test_settings import settings
+
+
+def generate_presigned_url_response(did, protocol="", status=200):
+    full_url = (
+        "https://fictitious-commons.io/data/download/" + did + "?protocol=" + protocol
+    )
+    presigned_url = {
+        "url": "https://storage.googleapis.com/nih-mock-project-released-phs123-c2/RootStudyConsentSet_phs000007.Whatever.v666.p1.c2.FBI-BMW-CIA.tar.gz?GoogleAccessId=internal-someuser-1399@dcpstage-210518.iam.gserviceaccount.com&Expires=1582215120&Signature=hUsgjkegdsfkjbsajkafnsdjksdnfjknbdsajkfbsdkjfbjdfbkjdasfbnjsdnfjsnd2FTr%2FKs2kGKs0fJ8v5elFk5NQAYdrGcU3kROrzJuHUbI%2BMZ839SAbAz2rbMBuC9e46%2BdB91%2FA==&userProject=dcf-mock-project"
+    }
+    responses.add(responses.GET, full_url, json=presigned_url, status=status)
+    return presigned_url
+
+
+def get_doc(
+    has_version=True, multiple_endpointurl=False, has_content=False, urls=list()
+):
+    doc = {
+        "form": "object",
+        "size": 123,
+        "urls": ["s3://endpointurl/bucket/key"],
+        "hashes": {"md5": "8b9942cf415384b27cadf1f4d2d682e5"},
+    }
+    if has_version:
+        doc["version"] = "1"
+    if multiple_endpointurl:
+        for url_type in ["gs", "ftp", "sftp"]:
+            doc["urls"].append("{}://endpointurl/bucket/key".format(url_type))
```

Comment:
the `urls` param is not used right now, it should be used to set the urls

my suggestion was to get rid of the `multiple_endpointurl` param so that you don't have to hardcode `://endpointurl/bucket/key` in this function, which lets the tests control which urls to use, and check the result more easily